nav2_dwb_controller/dwb_core/src/dwb_local_planner.cpp

```diff
@@ -66,7 +66,7 @@ DWBLocalPlanner::DWBLocalPlanner()
 
 void DWBLocalPlanner::configure(
   const rclcpp_lifecycle::LifecycleNode::SharedPtr & node,
-  const std::shared_ptr<tf2_ros::Buffer> & tf,
+  std::string /*name*/, const std::shared_ptr<tf2_ros::Buffer> & tf,
```

Comment:
What use do you see for the name?

Reply:
Multiple instances in the server so you can tell the difference between them